memcached/init.lua

```diff
@@ -115,7 +115,7 @@ local typetable = {
     expire_enabled = {
         'boolean',
         function() return true end,
-        function(x) return true end,
+        function() return true end,
```

Comment:
(If we'll keep the underscore, please, do here as well.)

Reply:
Added.